beam_sink/sinks/mysql.py

```diff
@@ -27,6 +39,19 @@ def expand(self, pcoll):
 
 
 class MySQLInsert(beam.PTransform):
+    """
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First line should end with a period, question mark, or exclamation point (not 'e')](https://app.codacy.com/manual/lislemitchell/beam-sink/pullRequest?prid=5069736)